### ISSUE TITLE
W&B Logging Multiple Images

### ISSUE THREAD

**Issue by Chadt54** [NVlabs/neuralangelo#131]
I see an another issue that in W&B there's these sliders at the bottom of the images where it seems like you can see a history/progression of images, but for some reason whenever I run Neuralangelo on my computer and connect it up to W&B it never shows these sliders, and only seems to show the very first image, and no more. 

Do you know what setting might be wrong somewhere to get more than a single image to show up? (so there's those sliders as shown in this example image below)

![270239842-2d479361-eee1-4e82-8842-bfeea5f2b8d0](https://github.com/NVlabs/neuralangelo/assets/1322965/a28662f1-1193-46c5-bb8f-7f978f40bb75)

**Comment by xsl316** [comment]
Have you solved it?

**Comment by Chadt54** [comment]
No, but I also stopped working on this project shortly after, so haven't spent much time on this since then.
